spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -240,17 +226,12 @@
 
       expect(attestation_response.attestation_type).to eq("Basic")
       expect(attestation_response.attestation_trust_path).to be_kind_of(OpenSSL::X509::Certificate)
+      expect(attestation_response.aaguid).to eq("00000000-0000-0000-0000-000000000000")
```

Comment:
This doesn't make sense to me currently 🤔 https://www.w3.org/TR/webauthn/#android-safetynet-attestation mentions nothing special about this being zeroed out, there's nothing in the attestation certificate either. I'll try capturing fresh data and see if something maybe changed on the Android/Play Services side.